modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -1,119 +1,80 @@
 
-The network plugin can provide additional features, like IPsec, that also must be decreased the MTU. For additional information, see the documentation.
+The network plugin can provide additional features, like IPsec, that also must be decreased the MTU.
+
+ifdef::local-zone[]
+For more information, see link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation.
+endif::local-zone[]
+ifdef::wavelength-zone[]
+For more information, see link:https://docs.aws.amazon.com/wavelength/latest/developerguide/how-wavelengths-work.html[How AWS Wavelength work] in the AWS documentation.
+endif::wavelength-zone[]
+ifdef::post-aws-zones[]
+You can access the following resources to learn more about a respective zone type:
+
+* See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation.
+
+* See link:https://docs.aws.amazon.com/wavelength/latest/developerguide/how-wavelengths-work.html[How AWS Wavelength work] in the AWS documentation.
+endif::post-aws-zones[]
 ====
 
-{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zones locations. Because of the type and size limitations of resources like EC2 and EBS on Local Zone resources, the default instance type can vary from the traditional worker pool.
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS {zone-type} locations. Because of the type and size limitations of resources like EC2 and EBS on {zone-type} resources, the default instance type can vary from the traditional worker pool.
 
-The default Elastic Block Store (EBS) for Local Zone locations is `gp2`, which differs from the regular worker pool. The instance type used for each Local Zone on edge compute pool also might differ from worker pools, depending on the instance offerings on the zone.
+The default Elastic Block Store (EBS) for {zone-type} locations is `gp2`, which differs from the regular worker pool. The instance type used for each {zone-type} on edge compute pool also might differ from worker pools, depending on the instance offerings on the zone.
 
-The edge compute pool creates new labels that developers can use to deploy applications onto AWS Local Zones nodes. The new labels are:
+The edge compute pool creates new labels that developers can use to deploy applications onto AWS {zone-type} nodes. The new labels are:
 
 * `node-role.kubernetes.io/edge=''`
+ifdef::local-zone[]
 * `machine.openshift.io/zone-type=local-zone`
+endif::local-zone[]
+ifdef::wavelength-zone[]
+* `machine.openshift.io/zone-type=wavelength-zone`
+endif::wavelength-zone[]
+ifdef::post-aws-zones[]
+* `machine.openshift.io/zone-type=local-zone`
+* `machine.openshift.io/zone-type=wavelength-zone`
```

Comment:
please don't duplicate the key `machine.openshift.io/zone-type=`. advice to use another approach in that mixed book.

Reply:
Yes, it gets complicatd with this few situations. Would the following work?

* Local Zones only: `machine.openshift.io/zone-type=local-zone`
* Wavelength Zones only: `machine.openshift.io/zone-type=wavelength-zone`